test/integration/smoke/test_network_traffic_type_api.py

```diff
@@ -0,0 +1,50 @@
+from marvin.cloudstackTestCase import cloudstackTestCase
+from marvin.lib.base import (
+    TrafficType,
+    PhysicalNetwork,
+    Zone,
+)
+from nose.plugins.attrib import attr
+
+
+class TestTrafficTypeApi(cloudstackTestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.test_client = super(TestTrafficTypeApi, cls).getClsTestClient()
+        cls.api_client = cls.testClient.getApiClient()
+        cls.services = cls.testClient.getParsedTestDataConfig()
+        cls._cleanup = []
+        cls.services["advanced_sg"]["zone"]["name"] = "TestTrafficTypeApi-zone"
+        cls.zone = Zone.create(cls.api_client, cls.services["advanced_sg"]["zone"])
+        cls._cleanup.append(cls.zone)
+
+        cls.physical_network = PhysicalNetwork.create(
+            cls.api_client,
+            cls.services["l2-network"],
+            isolationmethods="VLAN",
+            zoneid=cls.zone.id,
+        )
+        cls._cleanup.append(cls.physical_network)
```

Comment:
this will need a 
```suggestion
        cls._cleanup.append(cls.physical_network)

        @classmethod
        def tearDownClass(cls):
            super(TestTrafficTypeApi, cls).tearDownClass()
            
```
to work